lib/db/mysql.js

```diff
@@ -585,6 +585,60 @@ module.exports = function (log, error) {
       )
   }
 
+  MySql.prototype.multipleQueries = function (poolName, queries, finalQuery) {
+    var err
+    return this.getConnection(poolName)
+      .then(
+        function (connection) {
+          var results = []
+
+          return runQuery(0)
```

Comment:
You may be able to use `P.each` [1] here rather than making a custom function to run them in turn.

[1] https://github.com/petkaantonov/bluebird/blob/master/API.md#eachfunction-iterator---promise